inc/lightbox/upgrade.php

```diff
@@ -3,28 +3,37 @@
 
 if ( !defined( 'W3TC' ) )
 	die();
-
-
 ?>
 <div id="w3tc-upgrade">
-	<div class="w3tc-overlay-logo"></div>
-	<div class="w3tc_overlay_upgrade_header">
-		<iframe src="https://www.w3-edge.com/checkout-ad/?data_src=<?php echo esc_attr( $data_src ) ?>&client_id=<?php echo esc_attr( $client_id ) ?>" width="100%" height="420px"></iframe>
+
+	<div id="w3tc_upgrade_header">
+
+		<div>
+			<div>
+				<strong>W3 TOTAL CACHE</strong><br />
+				<span style="font-size:16px;"><?php esc_html_e( 'Unlock more performance options', 'w3-total-cache' ); ?></span>
+			</div>
+		</div>
+
+		<div>
+			<?php if ( true ): ?>
```

Comment:
button doesnt work on http:// websites, that's why there was a condition there

Reply:
Very nice catch @maxicus, that was my debug code in there... I fixed it in https://github.com/W3EDGE/w3-total-cache/pull/211/commits/c01bf9db1f8aadc7e5a27d69003f2a131f12d39c